Assets/Scripts/Entity/EntitySounds.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+
+namespace Entity
```

Comment:
Could put this into GameAudio

Reply:
Ah, didn't notice we have a namespace specifically for audio 👍